docs/asciidoc/modules/ROOT/pages/deprecations-and-additions/index.adoc

```diff
@@ -0,0 +1,515 @@
+----
+apoc.meta.cypher.types(node-relationship-map)
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.meta.isType(value,type)
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.meta.cypher.isType(value,type)
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.meta.typeName(value)
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.meta.cypher.type(value)
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.periodic.rock_n_roll_while('some cypher for knowing when to stop', 'some cypher for iteration', 'some cypher as action on each iteration', 10000) YIELD batches, total
+----
+a|
+Partially replaced in Cypher with:
+[source, cypher, role="noheader"]
+----
+CALL {} IN TRANSACTIONS OF n ROWS
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.periodic.rock_n_roll('some cypher for iteration', 'some cypher as action on each iteration', 10000) YIELD batches, total
+----
+a|
+Replaced in Cypher with:
+[source, cypher, role="noheader"]
+----
+CALL {} IN TRANSACTIONS OF n ROWS
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.create.vPattern({_labels:['LabelA'],key:value},'KNOWS',{key:value,...}, {_labels:['LabelB'],key:value}) returns a virtual pattern
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.create.virtualPath(['LabelA'],{key:value},'KNOWS',{key:value,...},['LabelB'],{key:value})
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.create.vPatternFull(['LabelA'],{key:value},'KNOWS',{key:value,...},['LabelB'],{key:value}) returns a virtual pattern
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.create.virtualPath(['LabelA'],{key:value},'KNOWS',{key:value,...},['LabelB'],{key:value})
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.xml.import(url, config)
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.import.xml(file,config)
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.refactor.cloneNodesWithRelationships([node1,node2,...])
+----
+a|
+Use the following instead, and set withRelationships = true:
+[source, cypher, role="noheader"]
+----
+apoc.refactor.cloneNodes(nodes, withRelationships, skipProperties)
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.text.phonetic(value) yield value
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.text.phonetic(text) yield value
+----
+
+a|
+label:procedure[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+apoc.text.doubleMetaphone(value) yield value
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+apoc.text.doubleMetaphone(text) yield value
+----
```

Comment:
The replacement is perhaps not super obvious to me because the replacement has the same name. Could there be possible to improve this a little?